app/models/movie.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 class Movie < ApplicationRecord
-  attribute :duration, :string
-  has_many :screenings
+  has_many :screenings, dependent: :destroy
```

Comment:
Consider case when you want to remove Movie that has Screenings with Reservations. You definitely don't want to do that